### DISCUSSION TITLE
Select with search no focus on input

### DISCUSSION THREAD

**Discussion by bombillazo** [ant-design/ant-design#51746] [Q&A]
Hello, on mobile we're having a snag when using search with the select component. the component automatically focuses on the search select and the keyboard comes up, making the experience weird when we have long list of options.

Is there any way to required tapping on the input again to make the keyboard come up?

**Comment by petercat-assistant** [comment]
在移动设备上使用 Ant Design 的 Select 组件时，默认行为是当组件被激活时，输入框会自动获得焦点并弹出键盘。如果你希望在首次点击时不弹出键盘，而是需要再次点击输入框才能弹出键盘，这可能需要一些自定义的解决方案，因为这并不是组件的默认行为。

你可以尝试以下方法来实现这个效果：

1. **自定义组件行为**：你可以通过监听 Select 组件的 `onFocus` 事件，并在首次点击时阻止默认行为。可以通过状态管理来记录是否是首次点击。

2. **使用 Ref 和原生事件**：通过使用 React 的 `ref` 来获取输入框的 DOM 元素，并手动控制其焦点行为。

3. **修改样式**：如果只是为了视觉上的效果，可以通过 CSS 来隐藏键盘，但这通常不是推荐的做法，因为它可能会影响用户体验。

如果这些方法都无法满足你的需求，可能需要对组件进行更深入的定制，或者查看社区是否有类似的解决方案或插件。

如果你有进一步的问题或需要代码示例，请告诉我！

如需进一步帮助，请在评论中描述您的问题并@petercat-assistant与我开始对话。
